PubHub/PubHub.Common/Services/BookService.cs

```diff
@@ -23,6 +23,11 @@ internal BookService(IHttpClientFactory clientFactory, string clientName) : base
 
         // TODO (JBN): Change to GUIDs instead of int when that has been updated.
 
+        /// <summary>
+        /// Calls the API enpoint to retrieve all books through the <see cref="BookInfoModel"/> filtered on the searchQuery.
+        /// </summary>
+        /// <param name="queryOptions">The query options that is requested.</param>
+        /// <returns>Returns a list of <see cref="BookInfoModel"/></returns>
```

Comment:
Can you remove the _Return_ from the `<returns>` summary, as it will show up as _Return Returns_ in summary view